grassroots-frontend/src/Features/PhoneCanvass/Logic/PhoneCanvassIdentityStore.ts

```diff
@@ -10,17 +10,19 @@ export interface PhoneCanvassCallerStore {
 export function createPhoneCanvassCallerStore(): StoreApi<PhoneCanvassCallerStore> {
   return createStore<PhoneCanvassCallerStore>()(
     devtools(
-      (set) => {
-        return {
-          caller: undefined,
-          setCaller: (caller: PhoneCanvassCallerDTO): void => {
-            set({ caller });
-          },
-        };
-      },
-      {
-        name: "phonecanvass-caller-store",
-      },
+      persist(
```

Comment:
persist uses local storage to make the identity of a caller persist across page loads.